src/views/v2/commerce/ticket/price.php

```diff
@@ -9,14 +9,45 @@
  *
```

Comment:
I feel like this template should now be broken up into several templates. Maybe have `not-on-sale-price`, `on-sale-price`, and `on-sale-regular-price` templates, too? Just a thought.

Reply:
Done, thanks for the suggestion. But you might need to retouch this while working on FE again. But the base templates should be good for now.